### ISSUE TITLE
Cannot resolve symbol 'Builder'

### ISSUE THREAD

**Issue by tbremer19** [chrisjenx/Calligraphy#109]
Hey, I'm trying to use Calligraphy in my app but it fails. I get the error "Cannot resolve symbol 'Builder' after adding the piece of code below the onCreate();...
![unbenannt](https://cloud.githubusercontent.com/assets/5941636/5600025/333f13c0-92d2-11e4-80f6-eaa67264d8f3.PNG)

Is this a common issue?
How can I solve it?

Thank you!

**Comment by mseroczynski** [comment]
Hey there. That's just a clue but you got nothing to lose - try editing your build.gradle (app's not project's):

repositories {
    mavenCentral()
    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
}

dependencies {
    compile 'uk.co.chrisjenx:calligraphy:2.0.0-SNAPSHOT'
}

**Comment by chrisjenx** [comment]
Appologies @tbremer19 `master` is the basis of 2.0.0. a few more fixes need to merged in before release.

**Comment by tng10** [comment]
Thanks @mseroczynski and @chrisjenx 

The section Dependency in README.md

```javascript
dependencies {
    compile 'uk.co.chrisjenx:calligraphy:2.0.0'
}
```
should be replaced by

```javascript
dependencies {
    'uk.co.chrisjenx:calligraphy:2.0.0-SNAPSHOT'
}
```

**Comment by tbremer19** [comment]
@chrisjenx no problem :smile: 
And @tng10 @mseroczynski thank you guys for the solution, I didn't know that there is a Snapshot :+1:

**Comment by chrisjenx** [comment]
Yeah sorry. Needed to merge in the 2.x branch with the snapshot. As I
wanted other stuff in there. want to fix `ViewStubs`  before 2.x is
released.

**Comment by chrisjenx** [comment]
Released 2.0.0 to nexus
